Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "NumItems", "type": "uint32" }
+      ]
+    },
+    "ObjSubmarineSeqCoordinatorSpawner": {
+      "parent": "ObjPirateSeqCoordinatorSpawner",
+      "fields": [
+      ]
+    },
+    "ObjICBMSpawner": {
+      "fields": [
+        { "name": "StartDist", "type": "float32" },
+        { "name": "Speed", "type": "float32" },
+        { "name": "StartTime", "type": "float32" },
+        { "name": "WaitTime", "type": "float32" },
+        { "name": "Speed1st", "type": "float32" },
+        { "name": "StartTime1st", "type": "float32" },
+        { "name": "WaitTime1st", "type": "float32" },
+        { "name": "Speed2nd", "type": "float32" },
+        { "name": "StartTime2nd", "type": "float32" },
+        { "name": "WaitTime2nd", "type": "float32" }
+      ]
+    },
+    "EnemyBeetonSpawner": {
+      "parent": "EnemyBaseSpawner",
+      "fields": [
+        { "name": "FirstState", "type": "EnemyBeetonSpawner::FirstState" },
+        { "name": "MessageType", "type": "EnemyBeetonSpawner::MessageType" },
+        { "name": "IsVisibleInMessageType", "type": "EnemyBeetonSpawner::IsVisibleInMessageType" },
+        { "name": "IsResurrect", "type": "bool" },
+        { "name": "AlivePoint", "type": "vector3" },
+        { "name": "IsMoveAttack", "type": "bool" },
+        { "name": "StartPos", "type": "vector3" },
+        { "name": "IsMoveAttackRotate", "type": "bool" },
+        { "name": "IsMoveResurrectRotate", "type": "bool" },
+        { "name": "MoveSpeed", "type": "float32" },
+        { "name": "IsFixDirection", "type": "bool" },
+        { "name": "RadiusActive", "type": "float32" },
+        { "name": "RadiusAttack", "type": "float32" },
+        { "name": "ShotSpeed", "type": "float32" },
+        { "name": "IntervalShot", "type": "float32" },
+        { "name": "IntervalAttack", "type": "float32" },
+        { "name": "IntervalWait", "type": "float32" },
+        { "name": "NumShotOneTime", "type": "uint8" },
+        { "name": "NumAttackOneTime", "type": "uint8" }
+      ]
+    },
+    "EnemyEggPawnNormalSpawner": {
+      "parent": "EnemyEggPawnBaseSpawner",
+      "fields": [
+      ]
+    },
+    "EnemyEggPawnGunSpawner": {
+      "parent": "EnemyEggPawnBaseSpawner",
+      "fields": [
+        { "name": "ShotType", "type": "EnemyEggPawnGunSpawner::ShotType" },
+        { "name": "AttackTime", "type": "float32" },
+        { "name": "AttackInterval", "type": "float32" },
+        { "name": "LimitDistance", "type": "float32" },
+        { "name": "NumAttackOneTime", "type": "uint8" },
+        { "name": "IntervalAttack", "type": "float32" }
+      ]
+    },
+    "EnemyEggPawnKnightSpawner": {
+      "parent": "EnemyEggPawnBaseSpawner",
+      "fields": [
+        { "name": "ShieldType", "type": "EnemyEggPawnKnightSpawner::ShieldType" },
+        { "name": "StartTime", "type": "float32" },
+        { "name": "AttackTime", "type": "float32" },
+        { "name": "AttackInterval", "type": "float32" }
+      ]
+    },
```

Comment:
Parameters named based on in-game functionality and other parameters with the same functionality from other objects.